### PR TITLE
Meta - 4.6.14.0.2

### PR DESCRIPTION
**Pull request by chauduyphanvu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The Chartboost Mediation Meta Audience Network adapter mediates Meta Audience Ne
 
 In your `build.gradle`, add the following entry:
 ```
-    implementation "com.chartboost:chartboost-mediation-adapter-meta-audience-network:4.6.14.0.1"
+    implementation "com.chartboost:chartboost-mediation-adapter-meta-audience-network:4.6.14.0.2"
```
Comment (cb-jpadilla) [inline README.md:16]: I don't think we need to change this in the README, do we? 🤔

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,9 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 4.6.14.0.2
```
Comment (cb-jpadilla) [inline CHANGELOG.md:6]: Can we amend this with the rest of the newer releases??

File: MetaAudienceNetworkAdapter/build.gradle.kts [hunk 3]
```diff
@@ -34,7 +34,7 @@ android {
         minSdk = 21
         targetSdk = 33
         // If you touch the following line, don't forget to update scripts/get_rc_version.zsh
-        android.defaultConfig.versionName = System.getenv("VERSION_OVERRIDE") ?: "4.6.14.0.1"
+        android.defaultConfig.versionName = System.getenv("VERSION_OVERRIDE") ?: "4.6.14.0.2"
```
Comment (cb-jpadilla) [inline MetaAudienceNetworkAdapter/build.gradle.kts:37]: We don't want this to be merged to main, but we do need it for release? Hmmm 🤔

File: README.md (hunk 1 above)
Comment (chauduyphanvu) [reply README.md:16]: Oh yeah good catch.

**Review by cb-jpadilla — COMMENTED** [review]
Code is good, we just have to think on the release. ✔️

File: CHANGELOG.md (hunk 2 above)
Comment (bichenwang) [reply CHANGELOG.md:6]: yeah, let's amend history. like slot it in below the 6.15.x but above the 4.6.14.0.1

File: MetaAudienceNetworkAdapter/build.gradle.kts (hunk 3 above)
Comment (bichenwang) [reply MetaAudienceNetworkAdapter/build.gradle.kts:37]: yeah, that's fine. let's leave it on its own branch. actually I guess it doesn't really matter as long as the tag is good. we shouldn't merge to main. that is correct. once we make the push, we do not merge this back into main.

File: README.md (hunk 1 above)
Comment (bichenwang) [reply README.md:16]: well it depends. if we want the tagged release to have this good readme history, then I guess?

**Review by bichenwang — COMMENTED** [review]
approving for release. please don't merge into main. thanks!

File: README.md (hunk 1 above)
Comment (cb-jpadilla) [reply README.md:16]: If we want to keep history, that can work. My prime concern was just the merging to main. 👍🏼

**Review by CB-RyanMcCormick — APPROVED** [review]

**Comment by chauduyphanvu** [comment]
Released. Closing.

**Pull request closed without merging**